Core/GUI/GameTooltipHooks.lua

```diff
@@ -538,14 +558,15 @@ hooksecurefunc(GameTooltip, "SetQuestLogItem", function(self, type, index)
 	local itemString = GetQuestLogItemLink(type, index)
 	processItemString(itemString)
 end)
+]]
 
 -- TOOLTIP: EMISSARY QUEST REWARD
 
 -- hooksecurefunc("GameTooltip_AddQuestRewardsToTooltip", function(self, questID)
---    if GetNumQuestLogRewards(questID) > 0 then
---        local _, _, _, _, _, id = GetQuestLogRewardInfo(1, questID)
---        if id then
---            processItem(id)
---        end
---    end
+--	if GetNumQuestLogRewards(questID) > 0 then
+--		local _, _, _, _, _, id = GetQuestLogRewardInfo(1, questID)
+--		if id then
+--			processItem(id)
+--		end
+--	end
```

Comment:
Was the format change done on purpose or did the autoformatter malfunction? (If it's the latter, I should look into why this happened...)

Reply:
No problem; I can remove it later. Generally, whatever stylua outputs is the desired format (to keep things simple).